src/app/store.ts

```diff
@@ -1,10 +1,15 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { TypedUseSelectorHook, useSelector } from 'react-redux';
 
-const rootReducer = combineSlices();
+import { filterSlice, todosSlice, currentTodoSlice } from '../features';
+
+const rootReducer = combineSlices(todosSlice, filterSlice, currentTodoSlice);
 
 export const store = configureStore({
   reducer: rootReducer,
 });
 
+export const selector: TypedUseSelectorHook<RootState> = useSelector;
```

Comment:
Rename selector to useAppSelector to follow React hook naming conventions and make its purpose clear
```suggestion
export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
```